x-pack/plugins/session_view/package.json

```diff
@@ -5,7 +5,7 @@
     "private": true,
     "license": "Elastic-License",
     "scripts": {
-        "test:jest": "node ../../scripts/jest",
+        "test:jest": "node ../../scripts/jest --coverage",
```

Comment:
think we could add another line for coverage since it's more time and resource-consuming:

```
"test:jest": "node ../../scripts/jest",
"test:coverage": "node ../../scripts/jest --coverage",
```